src/hotspot/share/adlc/formssel.cpp

```diff
@@ -25,6 +25,8 @@
 // FORMS.CPP - Definitions for ADL Parser Forms Classes
 #include "adlc.hpp"
 
+#define remaining_buflen(buffer, position) (sizeof(buffer) - (position - buffer))
```

Comment:
Consider additional parenthesis, e.g. `((position) - (buffer))`.

Reply:
It makes sense to me.  Thanks!